lib/log.js

```diff
@@ -0,0 +1,33 @@
+/*
+ Copyright Red Hat, Inc., and individual contributors
+
+ Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Sections of code should not be "commented out". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)